client/state/themes/actions.js

```diff
@@ -629,8 +629,8 @@ export function pollThemeTransferStatus( siteId, transferId, interval = 3000, ti
 				return resolve();
 			}
 			return wpcom.undocumented().transferStatus( siteId, transferId )
-				.then( ( { status, message, themeId } ) => {
-					dispatch( transferStatus( siteId, transferId, status, message, themeId ) );
+				.then( ( { status, message, uploaded_theme_slug } ) => {
```

Comment:
Initially I thought the field in the status response would be `themeId` but it will actually be `uploaded_theme_slug` when it gets deployed.